docs/source/R-api.rst

```diff
@@ -381,11 +381,11 @@ Extracts the ID of the run or experiment.
 Arguments
 ---------
 
-+------------+----------------------------------------------------+
-| Argument   | Description                                        |
-+============+====================================================+
-| ``object`` | An ``mlflow_run`` or ``mlflow_experiment`` object. |
-+------------+----------------------------------------------------+
+========== ==================================================
```

Comment:
These changes are due to a pandoc version difference. I have installed the most recent version locally. They result in no visual differences.